pandas/core/frame.py

```diff
@@ -3265,9 +3265,16 @@ def memory_usage(self, index: bool = True, deep: bool = False) -> Series:
             index=self.columns,
         )
         if index:
-            result = self._constructor_sliced(
-                self.index.memory_usage(deep=deep), index=["Index"]
-            ).append(result)
+            from pandas.core.reshape.concat import concat
+
+            result = concat(
+                [
+                    self._constructor_sliced(
```

Comment:
nitpick: can you do `whatever = self._constructor_sliced(...)` in a separate line